src/NzbDrone.Core/Download/Clients/QBittorrent/QBittorrentLabel.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Newtonsoft.Json;
+
+namespace NzbDrone.Core.Download.Clients.QBittorrent
+{
+    [JsonObject]
+    public class QBittorrentLabel
+    {
+        [JsonProperty("name")]
```

Comment:
PS: Json serializer automatically handles 'name' vs 'Name', so the JsonProperty attribute is redundant. (JsonObject isn't needed either)